{{ cookiecutter.project_slug }}/requirements.txt

```diff
@@ -3,3 +3,7 @@ jina[devel]
 transformers==3.0.2
 torch==1.5.1
 {%- endif %}
+
+{%- if cookiecutter.task_type | lower == 'cv' %}
+tensorflow >= 2.0
```

Comment:
This could be specific to ```tensorflow==2.0.1``` since we have issues for one specific version that >=2.0 would permit.

Reply:
LGTM!